src/types.ts

```diff
@@ -47,8 +47,10 @@ export interface RetailCalendarOptions {
   /**
    * If LeapYearStrategy.Restated, 53rd week will belong to last month in year. First week won't belong to any month.
    * If LeapYearStrategy.DropFirstWeek, 53rd week won't belong to any month in year. First week will belong to the first month.
+   * Note: restated: true is a deprecated option that is replaced by LeapYearStrategy.Restated
    */
-  leapYearStrategy: LeapYearStrategy
+  leapYearStrategy?: LeapYearStrategy
+  restated?: boolean
```

Comment:
Let's use [@deprecated](https://jsdoc.app/tags-deprecated.html) JSDoc tag for restated property. Popular IDEs support it and it will be [supported in Typescript 4.0](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/#deprecated-support).